pkg/gatherers/clusterconfig/io_config.go

```diff
@@ -0,0 +1,84 @@
+package clusterconfig
+
+import (
+	"context"
+
+	"github.com/openshift/insights-operator/pkg/config"
+	"github.com/openshift/insights-operator/pkg/record"
+	"github.com/openshift/insights-operator/pkg/utils/anonymize"
+)
+
+// GatherIOConfig gathers current configuration of the IO and the anonymized support secret
+//
+// * Location in archive: insights-operator/config/combined.json
+// * Location in archive: insights-operator/config/from_secret.json
+// * Location in archive: config/secrets/openshift-config/support/data.json
+// * Id in config: io_config
+// * Since version:
+//   * 4.12+
+func (g *Gatherer) GatherIOConfig(context.Context) ([]record.Record, []error) {
+	combinedConfig := g.configObserver.Config()
+	records := []record.Record{{
+		Name: "insights-operator/config/combined",
```

Comment:
The original goal was to gather just the support secret definition. I wouldn't complicate it with this "combined" data. Shouldn't be the values (when running in container) of the `data.json` and `combined.json` the same?

Reply:
I thought having the actual values may be more useful to whoever's gonna use the data for analysis. Should I delete `combined` and `from_secret` collectors?

Reply:
By default `data.json` won't exist because there's no support secret on the cluster, same applies to `from_secret.json` which is parsed configuration of the support secret. `combined.json` always exists and contains the actual state of config IO uses.

Reply:
When there's no support secret then the default (known) config is used right? Why to gather something (in each archive) that we know?

Reply:
Yeah there can be some differences in the various version, but we have it in Git right? It's basically the https://github.com/openshift/insights-operator/blob/master/config/pod.yaml so I would keep (in the archive) only the data coming from the `support` secret.

Reply:
changed it